lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,219 @@
+                and port.
+              type: str
+          kubeconfig:
+              description:
+              - Path to an existing Kubernetes config file. If not provided, and no other connection
+                options are provided, the OpenShift client will attempt to load the default
+                configuration file from I(~/.kube/config.json). Can also be specified via K8S_AUTH_KUBECONFIG
+                environment variable.
+              type: str
+          context:
+              description:
+              - The name of a context found in the config file. Can also be specified via K8S_AUTH_CONTEXT environment
+                variable.
+              type: str
+          host:
+              description:
+              - Provide a URL for accessing the API. Can also be specified via K8S_AUTH_HOST environment variable.
+              type: str
+          api_key:
+              description:
+              - Token used to authenticate with the API. Can also be specified via K8S_AUTH_API_KEY environment
+                variable.
+              type: str
+          username:
+              description:
+              - Provide a username for authenticating with the API. Can also be specified via K8S_AUTH_USERNAME
+                environment variable.
+              type: str
+          password:
+              description:
+              - Provide a password for authenticating with the API. Can also be specified via K8S_AUTH_PASSWORD
+                environment variable.
+              type: str
+          cert_file:
+              description:
+              - Path to a certificate used to authenticate with the API. Can also be specified via K8S_AUTH_CERT_FILE
+                environment variable.
+              type: str
+          key_file:
+              description:
+              - Path to a key file used to authenticate with the API. Can also be specified via K8S_AUTH_HOST
+                environment variable.
+              type: str
+          ssl_ca_cert:
+              description:
+              - Path to a CA certificate used to authenticate with the API. Can also be specified via
+                K8S_AUTH_SSL_CA_CERT environment variable.
+              type: str
+          verify_ssl:
+              description:
+              - "Whether or not to verify the API server's SSL certificates. Can also be specified via
+                K8S_AUTH_VERIFY_SSL environment variable."
+              type: bool
+          namespaces:
+              description:
+              - List of namespaces. If not specified, will fetch all virtual machines for all namespaces user is authorized
+                to access.
+              type: list
+          network_name:
+              description:
+              - In case of multiple network attached to virtual machine, define which interface should be returned as primary IP
+                address.
+              type: str
+          api_version:
```

Comment:
is this per connection or global? some options seem like they are overtly indented

Reply:
per connection, we have only one top level parameter `connections`